### ISSUE TITLE
Incorrect YAML highlighting

### ISSUE THREAD

**Issue by eevleevs** [ajaxorg/ace#4467]
Hello, I am getting an incorrect highlighting when a multi-word key follows a multi-line string.

expected:
```yaml
bill-to:  &id001
    street: |
        123 Tornado Alley
        Suite 16
    city name: East Centerville
```

actual (from kitchen sink):
![image](https://user-images.githubusercontent.com/5012744/110807548-70d0f080-8283-11eb-9d56-4e52c86766f2.png)

Note how "city name" has two different colors.

**Comment by AlexanderMisel** [comment]
A patch for yaml highlighter will fix this: https://github.com/AlexanderMisel/ace/tree/yaml-patch

**Comment by eevleevs** [comment]
Confirmed, thanks.

**Comment by andrewnester** [comment]
It's fixed in 1.15+ with this PR https://github.com/ajaxorg/ace/pull/4706
